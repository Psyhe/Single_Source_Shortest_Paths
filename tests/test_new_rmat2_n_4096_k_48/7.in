4096 602 687
602 2648 102
602 74 31
602 576 246
602 595 232
602 1754 189
602 592 24
602 2648 107
602 587 209
602 600 48
602 634 222
602 626 152
602 2682 101
602 74 131
602 665 72
602 24 81
602 3098 20
602 794 148
602 601 27
602 2803 235
603 2571 190
603 601 194
603 848 104
603 3210 31
603 72 189
603 2581 64
603 2779 29
603 779 212
603 1787 187
603 2635 161
603 752 254
603 2643 154
603 1601 189
603 1659 125
603 528 16
603 1266 224
603 26 111
604 3678 8
604 669 249
604 720 94
604 88 10
604 600 55
604 536 156
604 86 1
604 2200 79
604 2677 52
604 92 201
604 2572 112
604 568 232
604 328 101
604 2269 19
604 2653 74
604 3676 70
604 596 85
604 2582 228
604 980 22
604 1544 13
604 549 162
605 1117 1
605 1603 61
605 665 211
605 3769 2
605 577 125
605 733 130
605 221 247
605 661 27
605 637 145
605 596 129
605 601 160
605 845 174
605 2671 145
605 220 111
605 1657 126
606 350 22
606 581 12
606 788 86
606 2911 33
606 30 34
606 70 124
606 540 59
606 1386 76
606 989 10
607 1874 132
607 530 54
607 589 175
607 67 128
607 597 134
607 95 71
607 1663 85
607 590 226
608 2400 79
608 97 46
608 2163 36
608 736 187
608 1760 4
608 1656 2
608 832 90
608 2637 21
608 1641 60
608 612 11
608 736 53
608 2616 148
608 576 94
608 612 105
608 2592 140
608 576 188
608 624 10
608 320 38
608 624 173
608 2592 31
608 624 120
608 1652 249
608 836 175
608 564 248
608 2920 156
608 3008 218
608 800 19
608 1056 202
608 2608 187
608 1732 36
608 580 50
608 96 47
608 3680 15
608 2784 255
608 994 203
608 2656 62
608 2604 250
608 620 174
608 3584 172
608 118 183
608 801 240
609 753 215
609 544 41
609 629 74
609 1633 148
609 1645 217
609 881 169
609 737 97
609 2785 242
609 1923 70
609 2997 133
609 2593 101
609 615 43
609 2720 199
609 545 34
609 1539 40
609 614 245
609 755 133
609 1633 235
609 1858 93
609 2560 182
609 753 140
609 805 245
609 576 169
609 2093 80
609 97 146
609 112 137
609 513 44
609 704 180
609 993 230
609 612 241
609 705 15
609 576 40
609 2145 31
609 96 155
609 872 110
609 1289 65
609 625 144
609 741 217
609 617 161
610 102 138
610 1762 172
610 586 156
610 554 238
610 627 16
610 627 179
610 34 200
610 578 190
610 630 148
610 98 202
610 546 124
610 2803 89
610 650 24
610 706 26
610 2593 90
610 1059 16
610 99 143
610 839 63
610 1062 221
610 866 16
610 64 64
610 2674 3
610 550 87
610 98 37
610 738 234
610 66 249
610 864 176
610 106 6
610 2690 45
610 614 93
610 866 245
610 363 10
611 67 12
611 32 32
611 39 121
611 1121 245
611 1635 235
611 2054 195
611 355 60
611 753 216
611 870 21
611 1634 175
612 3668 159
612 102 55
612 101 23
612 36 4
612 2597 248
612 1656 1
612 100 225
612 804 230
612 101 247
612 2564 211
612 609 241
612 614 89
612 1636 106
612 2094 179
612 2624 157
612 628 239
612 1644 53
612 2628 22
612 3660 180
612 3060 138
612 228 66
612 622 185
612 2080 124
612 228 225
612 1188 145
612 68 122
612 608 11
612 1632 97
612 1108 241
612 608 105
612 10 71
612 821 99
612 70 246
612 580 111
612 1888 228
613 2684 45
613 3649 187
613 1879 78
613 637 236
613 37 178
613 533 52
613 2630 197
613 3718 102
613 1125 42
613 2597 212
613 2149 79
613 757 252
613 116 200
613 561 19
613 2372 189
614 876 222
614 1766 145
614 102 27
614 705 141
614 1088 142
614 358 215
614 609 245
614 867 56
614 878 254
614 3620 41
614 582 192
614 610 93
614 582 119
614 110 240
614 102 143
614 1644 245
614 612 89
614 2786 76
614 1910 112
614 1638 93
614 615 113
614 1062 212
614 358 201
614 870 246
614 227 153
614 2599 198
614 102 6
615 13 163
615 2679 14
615 743 214
615 614 113
615 103 140
615 578 193
615 1615 14
615 871 112
615 550 207
615 503 68
615 609 43
615 100 113
615 99 45
615 2887 32
616 1648 244
616 620 50
616 330 3
616 170 22
616 724 195
616 1656 179
616 57 141
616 416 21
616 746 80
616 620 242
616 1581 137
616 780 106
616 2760 84
616 43 248
616 1576 162
616 632 253
616 1768 220
616 741 152
616 2168 91
616 97 139
616 553 95
616 108 134
616 1834 144
616 738 58
616 888 140
616 874 44
616 1768 250
616 104 55
616 1640 183
616 3682 234
616 740 81
616 72 186
617 1547 44
617 677 72
617 585 40
617 621 201
617 626 65
617 73 122
617 1577 200
617 1633 161
617 193 113
617 1640 169
617 3629 37
617 2088 83
617 556 251
617 227 78
617 640 222
617 545 229
617 609 161
617 765 93
618 570 69
618 682 213
618 66 126
618 627 6
618 2726 54
618 2795 168
618 104 114
618 2671 124
618 2154 104
618 706 178
618 1642 124
618 2661 183
618 3619 159
618 120 124
618 1020 251
618 520 28
618 1002 14
618 744 50
619 632 206
619 579 228
619 539 244
619 571 199
619 2922 43
619 363 27
619 1608 62
619 635 40
620 616 242
620 588 191
620 608 174
620 520 67
620 45 167
620 874 131
620 1732 146
620 2223 43
620 1644 23
620 3661 123
620 556 22
620 108 255
620 3684 41
620 114 62
620 600 53
620 616 50
620 1638 104
620 280 203
620 548 51
620 1644 94
620 1637 135
620 1902 10
621 637 21
621 873 20
621 1647 238
621 352 129
621 1645 96
621 747 128
621 637 128
621 111 202
621 301 90
621 1644 202
621 617 201
621 713 165
621 1828 93
622 2022 10
622 1644 230
622 590 234
622 2794 34
622 2158 163
622 110 165
622 3046 44
622 612 185
622 2634 89
622 238 18
623 2683 82
623 38 80
623 1911 51
623 559 204
623 125 115
623 135 35
623 2665 71
624 720 35
624 1504 211
624 888 144
624 608 10
624 354 255
624 608 173
624 608 120
624 1586 215
624 2608 19
624 560 59
624 112 60
624 1620 151
624 2800 173
624 752 120
624 752 254
624 625 236
624 2672 135
624 884 189
624 593 50
624 2672 83
624 16 113
625 1649 108
625 115 97
625 3797 191
625 627 12
625 561 42
625 561 205
625 1761 250
625 609 144
625 1657 65
625 624 236
625 361 235
625 629 97
625 889 86
625 880 148
626 50 29
626 602 152
626 850 137
626 2628 22
626 2626 41
626 1334 155
626 562 63
626 617 65
626 1715 100
626 870 254
626 379 21
626 1150 248
626 1632 126
626 1400 134
626 1010 110
626 1010 175
626 561 252
626 1649 42
626 760 3
626 656 50
627 2677 189
627 2929 200
627 593 54
627 887 201
627 1019 112
627 625 12
627 610 16
627 2771 74
627 610 179
627 368 191
627 2291 111
627 618 6
628 884 165
628 612 239
628 1541 56
628 324 133
628 3060 12
628 2676 6
628 1747 255
628 116 41
628 247 134
628 1588 59
628 20 244
628 868 234
628 629 186
628 2168 124
628 597 144
629 549 47
629 609 74
629 628 186
629 625 97
629 1617 208
629 1651 250
629 231 6
629 573 230
630 3694 88
630 638 243
630 596 190
630 343 37
630 118 98
630 1650 45
630 711 64
630 631 62
630 2680 225
630 318 111
630 610 148
630 1620 23
631 567 100
631 885 94
631 2597 74
631 566 162
631 1703 182
631 1660 115
631 630 62
631 563 166
631 1591 112
631 542 253
632 2425 148
632 2603 246
632 124 193
632 216 197
632 616 253
632 1722 175
632 749 148
632 3696 49
632 104 123
632 732 46
632 619 206
632 112 131
632 636 1
632 568 172
632 937 98
632 121 104
632 2680 29
632 2682 212
632 760 9
632 1784 119
632 752 205
632 60 76
632 570 20
632 381 22
632 812 37
633 121 118
633 3249 13
633 17 101
633 2617 31
633 115 97
633 121 246
633 1905 243
633 2170 245
634 2642 217
634 3822 29
634 594 254
634 2835 121
634 1658 178
634 758 137
634 371 175
634 602 222
634 2651 17
634 2130 232
634 586 227
635 601 212
635 601 143
635 75 95
635 251 65
635 3323 11
635 1659 40
635 121 39
635 664 96
635 619 40
636 156 102
636 637 143
636 2168 192
636 570 243
636 2668 235
636 564 146
636 1139 95
636 1630 130
636 2104 17
636 2172 251
636 1742 39
636 632 1
636 120 233
636 2141 17
636 570 71
636 1656 51
637 621 21
637 2651 229
637 765 134
637 605 145
637 893 165
637 621 128
637 572 140
637 253 46
637 2744 195
637 2428 163
637 125 175
637 613 236
637 636 143
637 1149 86
638 1654 97
638 1790 124
638 630 243
638 2046 92
638 2942 229
638 852 207
638 14 141
638 574 70
639 189 25
640 617 222
640 704 171
640 2176 231
640 2564 228
640 832 222
640 1362 19
640 665 40
640 522 112
640 905 121
640 3716 133
640 2712 8
640 1792 238
640 1536 59
640 900 86
640 160 170
640 960 119
640 1024 185
640 644 135
640 752 99
640 896 199
640 960 241
640 1152 171
640 530 230
640 129 183
640 0 175
640 896 126
640 576 229
640 1760 62
640 768 222
640 65 186
640 641 130
640 650 152
640 548 123
640 2689 133
640 1672 98
640 704 124
640 132 63
640 705 230
640 672 123
640 944 197
640 682 10
640 192 29
640 656 220
640 704 3
640 709 54
640 674 222
640 34 82
640 897 217
640 2976 203
640 896 22
640 384 171
640 2560 41
640 936 50
640 2696 4
640 512 145
640 512 68
641 769 212
641 715 200
641 1028 230
641 1057 184
641 129 32
641 129 113
641 162 251
641 138 149
641 705 90
641 513 10
641 651 159
641 2048 145
641 2307 8
641 2049 161
641 129 240
641 545 177
641 672 110
641 513 238
641 1617 61
641 2049 202
641 128 255
641 585 109
641 2753 59
641 517 93
641 2186 149
641 645 173
641 129 233
641 1801 137
641 513 101
641 2697 44
641 706 39
641 9 7
641 657 204
641 2606 231
641 2177 177
641 640 130
641 385 189
641 667 150
641 521 93
641 1921 145
641 2209 248
641 657 144
641 1825 116
641 1795 226
641 2947 143
642 1664 113
642 514 78
642 682 150
642 2182 254
642 674 208
642 1538 140
642 658 187
642 514 95
642 2946 140
642 646 88
642 656 69
642 1994 23
642 1736 75
642 898 212
642 2594 78
642 130 232
642 648 180
642 595 21
642 1667 174
642 1747 245
642 914 226
642 522 1
642 515 42
642 138 229
642 2576 148
642 66 119
642 514 53
642 645 52
642 2688 170
642 2560 66
642 2 179
642 2114 60
642 915 64
642 142 41
643 995 119
643 647 115
643 129 19
643 3713 120
643 134 246
643 1281 63
643 2194 143
643 645 66
643 646 80
643 132 54
643 962 241
643 534 217
643 3735 10
643 515 61
643 2723 98
643 2433 83
643 512 205
643 657 238
643 735 200
643 2691 158
643 515 22
643 1153 80
643 714 114
644 772 1
644 652 118
644 660 20
644 2566 216
644 654 130
644 640 135
644 2692 18
644 896 137
644 2563 43
644 661 134
644 20 238
644 645 54
644 148 110
644 1696 137
644 2614 91
644 1749 24
644 2756 125
644 2944 8
644 165 99
644 652 30
644 652 91
644 149 46
644 1927 24
644 2695 61
644 2244 65
644 132 253
644 284 243
644 520 205
644 519 248
644 2692 32
644 772 223
644 709 252
644 132 152
644 252 16
644 677 88
644 516 102
644 668 92
644 1158 158
644 1668 88
644 110 32
644 645 138
644 902 65
644 708 203
645 197 150
645 545 16
645 149 231
645 2049 228
645 641 173
645 679 205
645 3297 152
645 644 138
645 140 48
645 133 40
645 781 133
645 649 100
645 1684 46
645 514 172
645 900 49
645 46 220
645 6 150
645 523 222
645 642 52
645 643 66
645 653 238
645 1673 176
645 165 63
645 644 54
645 3596 7
645 2690 104
645 165 145
645 257 226
645 659 47
645 2918 57
645 517 65
645 709 74
645 149 190
645 725 4
645 7 87
645 581 231
646 7 104
646 514 252
646 3718 243
646 2709 100
646 642 88
646 2966 177
646 902 30
646 694 140
646 2950 16
646 192 19
646 726 20
646 1670 67
646 710 255
646 2702 85
646 660 67
646 662 44
646 663 223
646 513 226
646 582 22
646 711 146
646 661 222
646 676 67
646 643 80
646 518 67
646 1665 22
646 2695 6
646 1668 141
647 902 212
647 1698 28
647 902 22
647 671 38
647 3982 53
647 643 115
647 902 92
647 2950 5
647 135 43
647 388 201
647 519 107
647 2695 132
647 134 138
648 141 68
648 650 59
648 776 121
648 1672 204
648 642 180
648 568 191
648 1176 157
648 520 211
648 169 68
648 2720 53
648 928 181
648 1672 6
648 2185 207
648 736 44
648 2588 94
648 776 70
648 910 127
648 2712 207
648 649 88
648 2736 240
648 905 121
648 200 118
648 154 154
648 1672 44
648 516 64
648 675 180
648 905 85
648 2562 212
648 778 231
648 664 62
648 3649 185
649 557 110
649 669 36
649 216 102
649 1737 199
649 1345 200
649 1665 185
649 1673 211
649 515 120
649 2761 241
649 697 171
649 461 26
649 905 177
649 137 153
649 648 88
649 653 58
649 1547 222
649 911 255
649 645 100
649 2633 110
649 653 55
649 2616 147
649 570 199
649 2232 31
650 138 221
650 706 99
650 2698 226
650 1960 7
650 554 8
650 174 17
650 1112 119
650 902 69
650 716 233
650 137 241
650 416 87
650 10 36
650 160 63
650 648 59
650 522 54
650 2762 1
650 966 226
650 906 211
650 640 152
650 2699 193
650 512 223
650 522 31
650 610 24
650 2203 205
650 654 254
650 1704 225
650 2566 31
650 520 79
650 922 71
650 667 10
651 15 212
651 235 80
651 2247 121
651 234 36
651 1595 245
651 711 253
651 683 47
651 803 111
651 581 92
651 905 63
651 3403 196
651 683 247
651 143 119
651 747 42
651 2376 109
651 1859 62
651 641 159
652 1929 11
652 653 8
652 1672 213
652 1197 28
652 204 101
652 672 140
652 684 130
652 904 181
652 708 110
652 2688 129
652 132 28
652 527 57
652 2366 237
652 397 1
652 2570 88
652 2568 103
652 644 118
652 141 209
652 142 46
652 684 91
652 2692 93
652 140 78
652 2693 117
652 132 99
652 2984 193
652 644 91
652 2780 127
652 937 79
652 738 190
652 140 58
652 1216 63
652 904 86
652 1547 77
652 644 30
653 1960 213
653 543 29
653 2701 212
653 713 111
653 157 83
653 1547 85
653 15 254
653 2709 33
653 645 238
653 3725 132
653 2185 68
653 593 46
653 649 58
653 897 181
653 516 143
653 649 55
653 384 164
653 29 83
653 453 41
653 1439 52
653 652 8
653 909 132
653 717 72
654 130 207
654 2590 206
654 670 113
654 650 254
654 430 228
654 678 69
654 2196 110
654 644 130
654 655 79
654 158 99
654 588 182
654 174 255
655 2719 114
655 715 173
655 807 165
655 719 79
655 519 14
655 2030 55
655 559 182
655 1671 246
655 899 174
655 687 22
655 927 210
655 2699 30
655 783 216
655 1679 226
655 135 127
655 1711 33
655 654 79
656 896 11
656 642 69
656 722 140
656 696 25
656 1937 209
656 544 153
656 667 238
656 52 163
656 657 236
656 528 100
656 440 229
656 756 32
656 65 248
656 2624 58
656 2068 132
656 17 100
656 403 63
656 3992 37
656 640 220
656 657 176
656 2576 233
656 626 50
656 512 7
656 1712 93
656 2577 173
656 596 232
656 528 208
656 2760 163
656 144 68
656 529 94
656 1168 243
656 2000 159
657 593 145
657 1681 244
657 772 226
657 721 184
657 2705 41
657 2803 126
657 704 5
657 1552 7
657 724 194
657 1617 32
657 947 144
657 705 42
657 673 30
657 641 204
657 1785 81
657 643 238
657 177 160
657 913 181
657 897 164
657 656 236
657 1689 254
657 2705 80
657 641 144
657 656 176
657 3219 37
657 533 213
657 661 118
657 2964 88
657 145 243
658 754 112
658 720 154
658 466 62
658 130 170
658 1686 7
658 690 53
658 2694 29
658 26 193
658 514 246
658 2586 228
658 2778 162
658 2242 235
658 538 47
658 1170 115
658 18 145
658 642 187
658 272 250
658 530 52
658 1682 103
658 2566 158
658 1682 221
658 912 94
658 688 75
658 784 226
658 534 96
658 542 51
658 594 30
658 1746 142
658 666 47
658 144 254
658 146 7
658 3794 255
659 147 182
659 667 224
659 153 60
659 663 63
659 665 93
659 2055 209
659 645 47
659 515 31
659 1681 205
659 2577 211
659 1682 91
659 539 163
659 272 96
659 730 96
659 19 240
659 219 82
659 146 80
659 2192 150
659 1009 167
660 212 185
660 646 67
660 532 223
660 1556 69
660 20 238
660 17 241
660 902 31
660 1668 156
660 1168 81
660 710 16
660 20 222
660 1692 198
660 788 9
660 916 227
660 156 251
660 0 177
660 748 70
660 516 17
660 534 233
660 2740 132
660 2964 163
660 80 216
660 644 20
660 2868 252
660 1684 200
660 662 254
660 788 19
660 1616 95
661 512 222
661 156 20
661 1558 20
661 148 41
661 2582 98
661 644 134
661 668 57
661 657 118
661 804 53
661 1425 41
661 529 84
661 200 232
661 646 222
661 541 78
661 541 70
661 2789 193
661 180 79
661 515 57
661 789 41
661 605 27
661 1941 224
661 2717 220
661 2821 226
661 1029 161
662 668 52
662 4 188
662 2055 82
662 22 40
662 950 160
662 951 207
662 1556 235
662 918 202
662 16 135
662 660 254
662 134 92
662 538 175
662 1798 82
662 670 64
662 646 44
662 946 149
662 2704 206
662 690 227
663 531 228
663 731 156
663 1683 103
663 695 12
663 23 45
663 671 3
663 646 223
663 519 188
663 659 63
663 151 163
664 696 185
664 908 216
664 3720 224
664 635 96
664 3728 171
664 665 198
664 536 119
664 312 184
664 2576 165
664 216 56
664 1433 101
664 2058 239
664 512 183
664 792 173
664 2968 200
664 904 207
664 1672 37
664 665 17
664 1624 106
664 3613 36
664 688 75
664 904 78
664 648 62
664 668 80
664 2712 230
664 680 111
665 138 18
665 1681 242
665 667 133
665 2649 247
665 2761 37
665 921 169
665 796 42
665 605 211
665 717 99
665 664 198
665 988 29
665 2777 197
665 681 195
665 659 93
665 699 37
665 602 72
665 145 92
665 137 182
665 408 118
665 568 79
665 184 93
665 667 213
665 1665 25
665 669 170
665 664 17
665 640 40
665 777 124
665 729 173
666 927 74
666 154 131
666 658 47
666 575 162
666 554 200
666 134 188
666 971 74
666 1009 119
666 1016 220
666 186 123
666 522 60
666 922 120
666 915 19
667 2585 75
667 1267 184
667 2681 56
667 665 133
667 1754 46
667 1425 64
667 671 70
667 1171 78
667 2202 253
667 728 76
667 671 164
667 675 45
667 656 238
667 641 150
667 25 18
667 83 178
667 659 224
667 2737 15
667 187 213
667 543 131
667 650 10
667 665 213
668 644 92
668 155 145
668 446 238
668 3742 153
668 2588 121
668 156 174
668 664 80
668 148 28
668 2782 77
668 140 11
668 661 57
668 923 167
668 21 108
668 150 245
668 669 27
668 662 52
668 3740 32
668 2780 25
668 222 199
668 920 180
669 729 66
669 649 36
669 909 71
669 604 249
669 143 110
669 668 27
669 532 119
669 727 38
669 2841 2
669 700 200
669 665 170
669 1948 59
669 2748 122
669 3640 93
670 926 129
670 390 15
670 2590 174
670 2718 175
670 191 9
670 654 113
670 730 178
670 922 8
670 569 134
670 47 195
670 158 217
670 725 128
670 150 169
670 1686 141
670 662 64
671 667 164
671 663 3
671 2718 107
671 779 199
671 543 123
671 734 149
671 2750 109
671 141 74
671 1663 59
671 647 38
671 667 70
672 641 110
672 568 173
672 2690 43
672 937 70
672 674 172
672 1840 174
672 1984 234
672 233 127
672 673 106
672 544 23
672 688 213
672 2256 7
672 772 49
672 1536 101
672 652 140
672 2596 250
672 678 93
672 788 30
672 168 192
672 132 137
672 2720 138
672 2721 150
672 997 229
672 164 19
672 32 240
672 1185 246
672 677 212
672 2592 62
672 640 123
672 1192 181
672 160 112
672 674 12
672 705 187
672 1954 171
672 1056 204
672 1193 134
672 0 109
672 1957 22
672 3 90
672 992 167
672 705 167
672 544 62
672 2784 100
672 2608 120
672 674 74
672 2689 102
672 436 241
672 674 253
672 673 32
672 944 94
672 224 1
673 177 64
673 933 115
673 936 45
673 672 32
673 549 16
673 2737 189
673 1025 87
673 544 144
673 681 204
673 710 20
673 681 5
673 937 54
673 512 63
673 1953 63
673 2725 86
673 744 161
673 672 106
673 1697 181
673 160 216
673 657 30
673 2595 6
673 769 243
673 1745 231
673 933 32
673 683 158
673 1761 130
673 161 1
673 1960 247
673 4001 233
673 1027 148
673 2729 21
673 1701 145
673 161 14
674 162 151
674 164 61
674 676 136
674 690 99
674 739 205
674 1248 147
674 34 133
674 3616 236
674 686 239
674 2722 205
674 640 222
674 934 114
674 1218 168
674 672 12
674 928 252
674 546 24
674 162 38
674 642 208
674 3754 146
674 678 80
674 3810 1
674 546 183
674 2791 219
674 672 253
674 114 244
674 34 0
674 672 74
674 994 105
674 1254 63
674 226 52
674 672 172
674 546 204
674 162 15
674 680 119
674 550 189
674 1578 127
674 676 159
674 1674 73
674 803 156
674 34 14
674 2726 101
674 2214 60
674 388 179
674 134 99
675 667 45
675 739 119
675 677 144
675 739 2
675 742 86
675 683 198
675 648 180
675 930 20
675 2755 58
675 687 217
675 1315 173
676 674 159
676 1668 59
676 164 181
676 646 67
676 2990 83
676 549 86
676 1184 29
676 1492 215
676 1670 117
676 3124 242
676 674 136
676 548 168
676 705 230
676 132 47
676 556 156
676 4028 206
676 2724 147
676 949 123
676 1709 124
676 680 156
676 1190 18
676 708 224
676 135 33
676 910 2
676 900 183
676 518 243
676 1702 210
677 143 62
677 1589 9
677 561 185
677 188 184
677 133 131
677 2087 59
677 164 54
677 1701 186
677 672 212
677 932 223
677 675 144
677 694 8
677 2733 58
677 644 88
677 901 127
677 2720 133
677 2613 175
677 617 72
678 931 142
678 3238 191
678 1690 213
678 166 213
678 672 93
678 546 198
678 965 236
678 2752 104
678 654 69
678 998 10
678 182 199
678 674 80
678 550 125
678 710 34
679 2213 15
679 1775 224
679 161 226
679 645 205
679 1189 206
679 551 140
679 711 101
679 903 96
679 2695 143
679 1671 99
679 2215 148
679 164 226
680 1144 75
680 440 137
680 1848 241
680 72 75
680 1188 213
680 812 170
680 952 101
680 676 156
680 0 89
680 512 122
680 674 119
680 739 128
680 2986 129
680 3752 10
680 1545 166
680 233 128
680 1576 154
680 664 111
680 3278 14
680 2730 138
680 159 44
680 688 75
680 713 83
680 2560 163
680 514 219
681 685 22
681 569 189
681 169 158
681 2464 90
681 2985 197
681 685 55
681 745 66
681 2184 198
681 553 67
681 1705 236
681 665 195
681 953 245
681 673 204
681 1248 43
681 673 5
681 745 89
681 1128 205
681 1057 172
681 3721 28
681 683 102
682 2699 167
682 640 10
682 43 75
682 618 213
682 2594 125
682 687 123
682 362 39
682 558 118
682 170 61
682 874 89
682 642 150
682 2216 160
682 1003 32
682 2607 248
683 1705 226
683 651 47
683 675 198
683 555 71
683 195 244
683 522 196
683 673 158
683 169 231
683 651 247
683 2747 124
683 555 142
683 681 102
683 3209 152
683 684 140
683 1736 188
684 1708 114
684 1707 240
684 524 106
684 252 152
684 940 183
684 1812 124
684 940 249
684 556 251
684 652 130
684 940 225
684 142 62
684 696 117
684 683 140
684 549 85
684 556 155
684 686 20
684 565 229
684 652 91
684 740 90
684 1550 166
685 429 247
685 681 55
685 13 201
685 2701 189
685 136 67
685 173 65
685 717 145
685 549 244
685 782 16
685 3004 180
685 941 0
685 681 22
685 2607 126
686 4000 154
686 495 172
686 674 239
686 684 20
686 804 153
686 188 115
686 2598 25
686 2590 75
686 558 180
686 235 105
687 682 123
687 702 61
687 675 217
687 31 9
687 2767 139
687 655 22
687 2701 231
