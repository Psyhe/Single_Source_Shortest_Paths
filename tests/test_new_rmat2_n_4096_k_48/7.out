38
35
44
60
41
82
10
52
46
31
7
60
7
90
46
53
67
83
56
74
68
57
20
74
45
62
40
41
81
104
38
82
106
82
37
95
147
132
32
27
47
31
42
34
39
58
62
50
47
79
28
36
114
39
22
34
38
56
39
33
77
79
54
37
85
57
25
52
74
82
6
16
13
47
49
66
45
65
50
21
42
123
92
43
72
40
